### DISCUSSION TITLE
Where do I find a compiler?

### DISCUSSION THREAD

**Discussion by VolvoBirkenstock** [mocaccinoOS/mocaccino#97] [General] [answered]
Hi, Mocaccino Gurus.  

If I want to compile an executable, what package do I use to specify a compiler?  
I do not see a gcc compiler or a gnu package containing a compiler within my installation.  
Do I need the docker package if I want to compile anything?  
If so, then I will need to learn about docker.  
Thank you very much in advance.

**Comment by joostruis** [comment, marked as answer]
`luet install development/gcc`

**Reply by VolvoBirkenstock** [reply]
Thank you, @joostruis.  I could not find the gcc package within [https://packages.mocaccino.org/mocaccino-os-commons/](https://packages.mocaccino.org/mocaccino-os-commons/), but after reading your response I modified the URL to [https://packages.mocaccino.org/](https://packages.mocaccino.org/), saw [https://packages.mocaccino.org/mocaccino-desktop-stable/](https://packages.mocaccino.org/mocaccino-desktop-stable/) and then looked inside, where I saw the gcc package.  

I may be a little slow, but I am learning.  :-)  THANK YOU!  

So, if I wanted to download the docker container and play around inside the container, would I need to get the container package from [https://packages.mocaccino.org/luet-official/](https://packages.mocaccino.org/luet-official/)?  Or is something more required?
